examples/worlds/auto_inertia_rolling_shapes.sdf

```diff
@@ -0,0 +1,206 @@
+    </light>
+
+    <model name="inclined_plane">
+      <static>true</static>
+      <pose>0 0 3 0 -0.52 0</pose>
+      <link name="link">
+        <collision name="collision">
+          <geometry>
+            <plane>
+              <normal>0 0 1</normal>
+              <size>100 100</size>
+            </plane>
+          </geometry>
+        </collision>
+        <visual name="visual">
+          <geometry>
+            <plane>
+              <normal>0 0 1</normal>
+              <size>100 100</size>
+            </plane>
+          </geometry>
+          <material>
+            <ambient>0.8 0.8 0.8 1</ambient>
+            <diffuse>0.8 0.8 0.8 1</diffuse>
+            <specular>0.8 0.8 0.8 1</specular>
+          </material>
+        </visual>
+      </link>
+    </model>
+
+    <model name="ground_plane">
+      <static>true</static>
+      <pose>0 0 0 0 0 0</pose>
+      <link name="link">
+        <collision name="collision">
+          <geometry>
+            <plane>
+              <normal>0 0 1</normal>
+              <size>100 100</size>
+            </plane>
+          </geometry>
+        </collision>
+        <visual name="visual">
+          <geometry>
+            <plane>
+              <normal>0 0 1</normal>
+              <size>100 100</size>
+            </plane>
+          </geometry>
+          <material>
+            <ambient>0.8 0.8 0.8 1</ambient>
+            <diffuse>0.8 0.8 0.8 1</diffuse>
+            <specular>0.8 0.8 0.8 1</specular>
+          </material>
+        </visual>
+      </link>
+    </model>
+
+    <model name="cylinder">
+      <pose>0 0 3.6 1.57079632679 0 0</pose>
+      <link name="cylinder_link">
+        <inertial>
+          <inertia>
+            <ixx>14.583333</ixx>
+            <ixy>0</ixy>
+            <ixz>0</ixz>
+            <iyy>14.583333</iyy>
+            <iyz>0</iyz>
+            <izz>12.5</izz>
+          </inertia>
+          <mass>100.0</mass>
+        </inertial>
+        <collision name="cylinder_collision">
+          <density>7810.0</density>
+          <geometry>
+            <cylinder>
+              <radius>0.5</radius>
+              <length>1.0</length>
+            </cylinder>
+          </geometry>
+        </collision>
+
+        <visual name="cylinder_visual">
+          <geometry>
+            <cylinder>
+              <radius>0.5</radius>
+              <length>1.0</length>
+            </cylinder>
+          </geometry>
+          <material>
+            <ambient>1 0 0 1</ambient>
+            <diffuse>1 0 0 1</diffuse>
+            <specular>1 0 0 1</specular>
+          </material>
+        </visual>
+      </link>
+    </model>
+
+    <include>
+      <name>hollow_cylinder</name>
+      <pose>0 -1.5 3.6 1.57079632679 0 0</pose>
+      <uri>https://fuel.gazebosim.org/1.0/jasmeetsingh/models/Hollow Cylinder</uri>
+    </include>
+
+    <model name="sphere">
+      <pose>0 1.5 3.6 0 0 0</pose>
+      <link name="sphere_link">
```

Comment:
the idea is to use `<inertial auto="true" />` in all of them or just in some ?

Reply:
Oh, thanks for pointing this out. The idea was to use `<inertia auto="true" />` in all. I have updated the world in [8ed75d7](https://github.com/gazebosim/gz-sim/pull/2119/commits/8ed75d7d086490812f4afa5478f6496628771728)